common-types/index.d.ts

```diff
@@ -81,3 +81,8 @@ interface TeamEvent {
   readonly attendees: IdolMember[];
   readonly uuid: string;
 }
+
+interface EventProofImage {
+  readonly url: string;
+  readonly fileName: string;
+}
```

Comment:
Hmm this is the same as the `ProfileImage` type... Does it make sense to make these the same type?

Reply:
I am not sure what fields would be needed for frontend integration other than these two. In that case, should I just combine both types together, or if there's a better way of doing it?

Reply:
Also, code for image processing is similar for both profile images and event images so I think that can be refactored out once the team events are integrated

Reply:
```
 type TeamEventAttendance = {
  memberEmail: string;
  teamEventName: string;
  hoursAttended?: string;
  image: string;
};
```

The way I implemented it on the frontend was when the form for requesting a team event credit is submitted, an object of type TeamEventAttendance is sent to the backend. I feel like information should be attached to the image, like the name of the team event name, hours attended (if it applies), etc. Maybe we can make the image attribute of the type TeamEventAttendance be an object that contains the url and filename? Guess we can figure it out when we integrate the frontend and backend